instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/ExperimentalConfig.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.config;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public final class ExperimentalConfig {
+
+  // lazy initialized, so that javaagent can set it, and library instrumentation can fall back and
+  // read system properties
+  @Nullable private static ExperimentalConfig instance = null;
```

Comment:
That's not true for this class - I think we can safely use a `static final` instance. It is `Config` that needs to have lazy initialization, because it's "real" instance is being set by the agent.